.scripts/cli.sh

```diff
@@ -9,7 +11,7 @@ rm -rf release
 mkdir -p release
 
 gox -output="release/tex-cli-$(git describe --tags)-{{.OS}}-{{.Arch}}" \
-    -ldflags "-X main.Version=$RELEASE -X main.PublicKey=$PUBLIC_KEY -s -w" \
+    -ldflags "-X main.Version=$VERSION -X main.CompileDate=$DATE -s -w" \
```

Comment:
Why was the public key removed? was it not used anyway?

Reply:
yea it's only needed in TemporalX's server CLI since that has a licensing components, whereas this is just the SDK.